hydradx/model/amm/omnipool_amm.py

```diff
@@ -67,7 +67,7 @@ def __init__(self,
         self.tvl_cap = tvl_cap
         self.stablecoin = preferred_stablecoin
         self.fail = ''
-        self.sub_pools = {}  # require sub_pools to be added through create_sub_pool
+        self.sub_pools = dict()  # require sub_pools to be added through create_sub_pool
```

Comment:
I'm confused why there are changes to omnipool_amm.py in a PR for reorganizing tests?

Reply:
That probably should not be in here, but the only difference it makes is cosmetic - for some reason with {}, PyCharm believes it is a dict of Oracles. With dict(), it (somehow) understands that we are talking about subpools. Either way it has no effect on functionality. For that reason I propose leaving it here.

Reply:
Okay, let's leave it.